helix-view/src/document.rs

```diff
@@ -1720,6 +1736,14 @@ impl Document {
         &self.selections
     }
 
+    pub fn view_data(&self, view_id: ViewId) -> Option<&ViewData> {
+        self.view_data.get(&view_id)
+    }
+
```

Comment:
Rather than unwrapping at all of the callsites of this function, let's `expect` here and return a `&ViewData`

Reply:
Done